containerd-shim-spin/src/engine.rs

```diff
@@ -43,6 +43,9 @@ const OCI_LAYER_MEDIA_TYPE_WASM: &str = "application/vnd.wasm.content.layer.v1+w
 /// Expected location of the Spin manifest when loading from a file rather than
 /// an OCI image
 const SPIN_MANIFEST_FILE_PATH: &str = "/spin.toml";
+/// Known prefix for the Spin application variables environment variable
+/// provider: https://github.com/fermyon/spin/blob/436ad589237c02f7aa4693e984132808fd80b863/crates/variables/src/provider/env.rs#L9
+const SPIN_APPLICATION_VARIABLE_PREFIX: &str = "SPIN_VARIABLE";
```

Comment:
I am not sure it is worth bringing in the variables crate for this constant (if we did make it pub). Maybe with the spin factors work, this will be more directly configured by the shim